src/Tokenizer/Analyzer/Analysis/EnumAnalysis.php

```diff
@@ -20,12 +20,12 @@
 final class EnumAnalysis extends AbstractControlCaseStructuresAnalysis
 {
     /**
-     * @var CaseAnalysis[]
+     * @var list<CaseAnalysis>
```

Comment:
👎🏻 I don't see the value of this change, tbh

Reply:
Static analysis tools interpret `CaseAnalysis[]` as `array<array-key, CaseAnalysis>`, which tells you nothing about keys. This change makes it easier to understand what do they actually mean:

- `array<string, CaseAnalysis>` means that keys are strings and probably are meaningful, maybe case name?
- `array<int, CaseAnalysis>` means that keys are integers and may mean something, maybe index of `T_CASE` token?
- `list<CaseAnalysis>` means that this is just a list, keys are not meaningful,
- `CaseAnalysis[]` may be any of the above.

Take a look at changes in `FunctionsAnalyser`. This method also uses `Foo[]` type, but turns out that in that case keys are actually argument names:

```diff
  /**
-  * @return ArgumentAnalysis[]
+  * @return array<string, ArgumentAnalysis>
   */
  public function getFunctionArguments(Tokens $tokens, int $functionIndex): array
```